openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -95,7 +95,11 @@ def update_translatable_fields(cr):
                     bool_or(imd.noupdate) AS noupdate
                 FROM ir_translation it
                 LEFT JOIN ir_model_data imd ON imd.model = %(model)s AND imd.res_id = it.res_id
-                WHERE it.type = 'model' AND it.name = %(name)s AND it.state = 'translated'
+                WHERE it.type = 'model'
+                    AND it.name = %(name)s
+                    AND it.state = 'translated'
+                    AND it.value IS NOT NULL
+                    AND length(it.value) > 0
```

Comment:
Shouldn't this be this way?

```suggestion
                    AND it.value != ""
```

Reply:
Hi, you can simplify the 2 conditions (not null + != '') with:
```suggestion
                    AND COALESCE(it.value, '') != ''
```

Reply:
Yes, this way can be more compact. In terms of performance, I think it's more or less the same. @etobella do you change it?

Reply:
@etobella do you perform this last change?

Reply:
Sure! Changes applied.